quest/__init__.py

```diff
@@ -6,10 +6,15 @@
 """
 from __future__ import (absolute_import, division,
                         print_function, unicode_literals)
-from builtins import *
 import pbr.version
-import os
+import warnings
 import logging
+import urllib3
+import os
+
+warnings.filterwarnings("ignore", message="numpy.dtype size changed")
+warnings.filterwarnings("ignore", message="numpy.ufunc size changed")
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
These InsecureRequestWarnings are coming from the user providers and I think we need to handle them there rather than disabling warnings in general.